server/models/db/add-worker-timeslot.js

```diff
@@ -0,0 +1,18 @@
+const _ = require('lodash');
```

Comment:
I would rename this file to be `schedule` or `timeslot` or another noun rather than `add-worker-timeslot` to increase flexibility in the future as other worker timeslot actions, such as viewing or fetching specific timeslots may be performed in the future and should be located within the same file for simplicity.

Reply:
Understood. Scability is important.